src/ogg-opus-decoder/types.d.ts

```diff
@@ -0,0 +1,25 @@
+type OpusDecodedAudio = {
+  channelData: Float32Array[];
+  samplesDecoded: number;
+  sampleRate: 48000;
+};
+
+declare module 'ogg-opus-decoder' {
+  export class OggOpusDecoder {
+    ready: Promise<void>;
+    reset: () => Promise<void>;
+    free: () => void;
+    decode: (data: Uint8Array) => OpusDecodedAudio;
+    decodeFrame: (data: Uint8Array) => OpusDecodedAudio;
+    decodeFrames: (data: Uint8Array) => OpusDecodedAudio;
```

Comment:
This class and the web worker class don't have a `decodeFrame` or `decodeFrames` method. Could you remove these?